src/G2/Execution/Reducer.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
Same here.  Do we use this?

Reply:
Yes, Line 405
https://github.com/BillHallahan/G2/pull/93/files/9386ddffd23b2029dbc9986236f21df618d13e72#diff-a99355b082cb03eb4c65ea6d05b10a3cb48cd384dd5d0e34c29faa0f657d08fbR405